scripts/spack/configs/blueos_3_ppc64le_ib_p9/spack.yaml

```diff
@@ -278,6 +278,34 @@ spack:
       - spec: zlib@1.2.7
         prefix: /usr
 
+    # External dependencies for opencascade
+    # Couldn't find the exact versions for the libx* libraries, so copied from toss4 versions
+    mesa:
+      buildable: False
+      externals:
+      - spec: mesa@18.3.4  # found via: glxinfo | grep -i version
+        prefix: /
```

Comment:
@white238 -- the opengl libraries were in `/lib64`, so I made the prefix `/` instead of `/usr`.
Is that the correct prefix? I tested this w/ one of our blueos configs and it worked.

Reply:
Wierd but correct? /shrug